src/utils/randomizer.js

```diff
@@ -59,7 +59,7 @@ function createKingdoms(numberOfPlayers, selectedCards) {
   const kingdomCardNames = [
     ...selectedCards,
     ...shuffle(
-      KINGDOM_CARDS.filter(card => !selectedCards.includes(card))
+      KINGDOM_CARDS.filter(card => !selectedCards.includes(card.name))
```

Comment:
does this even work? `KINGDOM_CARDS` is set to `Object.keys(BASE_CARDS)` which would be an array of strings